src/v2/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -157,22 +158,33 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
     const selectedBid = Number(values.selectedBid)
 
     if (requiresPaymentInformation) {
+      const stripeAddress = toStripeAddress(values.address)
+      const { phoneNumber } = values.address
+      const { setFieldError, setSubmitting } = actions
+
       try {
-        const { address } = values
-        const stripeAddress = toStripeAddress(address)
         const { error, token } = await stripe.createToken(stripeAddress)
 
         if (error) {
-          actions.setFieldError("creditCard", error.message)
-          actions.setSubmitting(false)
+          setFieldError("creditCard", error.message)
+          setSubmitting(false)
           return
         }
 
-        await createCreditCardAndUpdatePhone(
-          environment,
-          address.phoneNumber,
-          token.id
-        )
+        const { id } = token
+        const {
+          createCreditCard: { creditCardOrError },
+        } = await createCreditCardAndUpdatePhone(environment, phoneNumber, id)
+
+        // TODO: We are not handling errors for `updateMyUserProfile`. Should we?
```

Comment:
good todo – not sure myself yet, but think it's worth having as a note in the codebase 👍